src/sync/outgoingSyncUtils.js

```diff
@@ -222,6 +222,7 @@ const generateSyncData = (settings, recordType, record) => {
         paymentTypeID: record?.paymentType?.id ?? '',
         entry_time: getTimeString(record.entryDate),
         Date_order_written: getDateString(record.entryDate),
+        isCancellation: String(record?.isCancellation ?? false),
```

Comment:
this one should be 
```suggestion
        is_cancellation: String(record?.isCancellation ?? false),
```

Reply:
twice though, I think is very impressive


ad0086e